modules/projects-data-source/README.md

```diff
@@ -15,11 +20,7 @@ module "my-org" {
 }
 
 output "projects" {
-  value = module.my-org.projects
-}
-
-output "folders" {
-  value = module.my-org.folders
+  value = module.my-org.projects_numbers
```

Comment:
this looks like a typo (`project_numbers`)

Reply:
Ohh, good catch. Fixed now